templates/landscape/managed.html

```diff
@@ -15,250 +17,272 @@
-                      <td>N/A</td>
-                      <td>$9,470</td>
-                    </tr>
-                  </tbody>
-                </table>
+              <div class="p-logo-section__item">
+                {{ image(url="https://assets.ubuntu.com/v1/bd83cd83-opensearch-logo.png",
+                                alt="OpenStack",
+                                width="81",
+                                height="209",
+                                hi_def=True,
+                                loading="lazy",
+                                attrs={"class": "p-logo-section__logo"}) | safe
+                }}
+              </div>
+              <div class="p-logo-section__item">
+                {{ image(url="https://assets.ubuntu.com/v1/7589afda-vmware-logo.png",
+                                alt="VMWare",
+                                width="216",
+                                height="209",
+                                hi_def=True,
+                                loading="lazy",
+                                attrs={"class": "p-logo-section__logo"}) | safe
+                }}
               </div>
             </div>
           </div>
         </div>
-      </section>
+      </div>
+    </div>
+  </section>
 
-      <section class="p-strip is-white" style="background: white;">
-        <div class="row">
-          <div class="col-9 col-start-large-4 col-medium-4 col-start-medium-3 p-strip">
-            <h2>Got questions?</h2>
-            <p>Landscape's API can bridge any requirements our dashboard may not natively address.</p>
-            <p>
-              <a href="/landscape/managed#get-in-touch">Contact us about your unique requirements</a>
-            </p>
-          </div>
+  <section class="p-section">
+    <hr class="p-rule is-fixed-width" />
+    <div class="p-section--shallow">
+      <div class="row--50-50">
+        <div class="col">
+          <h2>Bring your own identity and access management platform</h2>
+        </div>
+        <div class="col">
+          <p>
+            Landscape supports OpenID Connect (OIDC) to authenticate users, and integrates enterprise identity providers like Azure AD, Google, Okta, IBM, and more.
+          </p>
+        </div>
+      </div>
+    </div>
+    <div class="u-fixed-width u-hide--small">
+      {{ image(url="https://assets.ubuntu.com/v1/9423d6a0-bring-own-identity.png",
+            alt="",
+            width="2464",
+            height="1028",
+            hi_def=True,
+            loading="lazy",
+            attrs={"class": "p-image-container--full-width is-highlighted"}) | safe
+      }}
+    </div>
+  </section>
+
+  <section class="p-section">
+    <hr class="is-fixed-width" />
+    <div class="row--50-50">
+      <div class="col">
+        <h2>
+          Mirror Ubuntu repositories,
+          <br />
+          and distribute software internally
+        </h2>
+      </div>
+      <div class="col">
+        <div class="u-sv3">
```

Comment:
I think this is needed, it also mis-aligns the text with the heading
Please look for other similar cases as well.
![image](https://github.com/user-attachments/assets/818c6768-5cc2-423d-90fd-5135841f6eed)

Reply:
Thanks for the careful observation Madhur!
However, the u-sv3 is needed to add a bit of spacing between the texts and the image. It doesn't affect the alignment of text with heading.

With u-sv3
![image](https://github.com/user-attachments/assets/c1dfe537-7011-4924-b0c7-a1bcfbbad280)


Without u-sv3
![image](https://github.com/user-attachments/assets/78553612-3840-4320-81c7-cfad1b6dcc80)

Reply:
Yes, you are right, I was thinking something else!